app/models/invenio_rdm_record_converter.rb

```diff
@@ -133,42 +135,50 @@ def creators(creators)
       # Galter Health Sciences Library is a User on production DigitalHub, but it does NOT have a formal_name value
       dh_user = User.find_by(formal_name: creator)
 
-      # organization
+      # Organization
       if organization?(creator)
-        name = creator
-        creator_type = "organisational"
-        family_name = ""
-        given_name = ""
-        identifiers = {}
-        affiliations = [] # only used for creator_type "personal"
-      # user
+        {
+          "person_or_org":
+            Hash.new.tap do |hash|
+              hash["name"] =  creator if creator.present? && !creator.include?("not identified")
+              hash["type"] = "organisational"
+            end
+        }.with_indifferent_access
```

Comment:
For some reason the keys were coming back as a mix of symbols and strings which was causing annoying issues with the spec for this function. The to_json spec is passing with this too so I think it's worth keeping in.